autosize-input.js

```diff
@@ -32,8 +32,20 @@
     // Apply the `font-size` and `font-family` styles of `elem` on the
     // `ghost` element.
     var elemStyle = window.getComputedStyle(elem)
-    var elemCssText = 'font-family:' + elemStyle.fontFamily +
-                     ';font-size:' + elemStyle.fontSize
+    var elemCssText = 'letter-spacing:' + elemStyle.letterSpacing +
+                     ';font-feature-settings:' + elemStyle.fontFeatureSettings +
+                     ';font-family:' + elemStyle.fontFamily +
+                     ';font-kerning:' + elemStyle.fontKerning +
+                     ';font-size:' + elemStyle.fontSize +
+                     ';font-stretch:' + elemStyle.fontStretch +
+                     ';font-style:' + elemStyle.fontStyle +
+                     ';font-variant:' + elemStyle.fontVariant +
```

Comment:
I know it seems silly and overkill, but one ligature or alternate number in a font could throw off the width calculation.